projects/chaingpt/index.js

```diff
@@ -44,11 +44,65 @@ const config = {
   ],
   era: [
     { factory: '0xf25F7c9522cdCD839697F1644CFCA1312306885C', fromBlock: 37458600 },
+  ],
+  blast: [
+    { factory: '0xBB9667bcbd04bD4F2C1f2FCd14995Bd3d00c5655', fromBlock: 8619958 },
+  ],
+  sonic: [
+    { factory: '0x9840652DC04fb9db2C43853633f0F62BE6f00f98', fromBlock: 11999667 },
+  ],
+  berachain: [
+    { factory: '0xBB9667bcbd04bD4F2C1f2FCd14995Bd3d00c5655', fromBlock: 787656 },
+  ],
+
+}
+
+const giveawayConfigs = {
```

Comment:
TVL coming from giveaways should be exported under key 'vesting' rather that 'tvl' because the tokens are non-circulating